addon/src/test-support/index.js

```diff
@@ -1,16 +1,28 @@
-import { getContext } from '@ember/test-helpers';
+import { importSync } from '@embroider/macros';
 import { resolve } from 'rsvp';
 import { run } from '@ember/runloop';
-import { Color } from 'ember-animated/color';
-import { relativeBounds } from 'ember-animated/-private/bounds';
-import { cumulativeTransform } from 'ember-animated/-private/transform';
+import { Color } from '../color';
+import { relativeBounds } from '../-private/bounds';
+import { cumulativeTransform } from '../-private/transform';
 import TimeControl from './time-control';
 
 export { TimeControl };
 export { default as MotionTester } from './motion-tester';
 
+// Re-export to ensure "instanceof" works properly within MotionTester.
```

Comment:
@ef4 and this is second part of the trick to make sure we use same entry-point for the code used in tests that involve `instanceof` which is a workaround for https://github.com/ef4/ember-auto-import/issues/503